www/index.html

```diff
@@ -468,10 +468,13 @@ <h3>Display settings</h3>
                                     <select class="form-control" id="appThemeSelect">
                                         <option value="light">Light</option>
                                         <option value="dark">Dark (app only)</option>
-                                        <option value="dark_invert">Dark + invert (generic)</option>
-                                        <option value="dark_mwInvert">Dark + invert (Wikimedia)*</option>
+                                        <option value="dark_invert">Dark (generic invert)</option>
+                                        <option value="dark_mwInvert">Dark (wikimedia invert)*</option>
+                                        <option class="auto" value="auto">Auto (app only)</option>
+                                        <option class="auto" value="auto_invert">Auto (generic invert)</option>
+                                        <option class="auto" value="auto_mwInvert">Auto (wikimedia invert)*</option>
```

Comment:
`class=auto` in case, browser doesn't support `prefers-color-scheme` so is used to remove those options from the DOM